documentation/staking-governance.md

```diff
@@ -13,21 +13,21 @@ This design with selectable locking duration helps to align token holders with t
 
 The mechanism for locking tokens works as follows: 
 
-- Users can lock tokens for a specific locking duration (between 2 weeks and 2 years). After the locking duration ends, the users can redeem their tokens.
-- Users can modify their locking positions (increase amount, extend locking duration) at any time.
-- Users have the option to pause their locking period countdown. That means, the remaining locking duration remains fixed until the user decides to resume it. Note that this results in higher voting power and staking rewards as described below. 
+- Users can **lock** tokens for a specific locking duration (between 2 weeks and 2 years). After the locking duration ends, the users can redeem their tokens.
+- Users can **modify** their locking positions (increase amount, extend locking duration) at any time.
+- Users have the option to **pause** their locking period countdown. That means, the remaining locking duration remains fixed until the user decides to resume it. Note that this results in higher voting power and staking rewards as described below. 
```

Comment:
In the "described below" we could add a link to the subsection on paused positions.

Reply:
done